readme.md

```diff
@@ -9,7 +9,8 @@ automatically on boot (no need to enter desktop mode).
 1. Clone this repo to your Deck.
 2. Run `sudo bash tailscale.sh` to install Tailscale (or update the existing
    installation).
-3. Run `sudo tailscale up --qr --operator=deck --ssh` to have Tailscale generate
+3. Run `source /etc/profile.d/tailscale.sh` to put the binaries in your path

```

Comment:
I don't think line 70 actually applies to the parent bash session. I think this is because we are executing in a new bash session via the `bash` command, but I'm not sure. Thus on first run I just added the helper telling users to either restart the session or load in the file manually. 

While line 70 does update the path for the rest of the script, we don't actually make use of it so it is superfluous. I can remove it if you'd like.

Reply:
> I don't think line 70 actually applies to the parent bash session

Ah good catch!

>While line 70 does update the path for the rest of the script, we don't actually make use of it so it is superfluous. I can remove it if you'd like.

I'm okay to keep it in case we want to add things in the future.